### DISCUSSION TITLE
Why is the background in "black on white" quite a dark grey?

### DISCUSSION THREAD

**Discussion by halloleo** [saulpw/visidata#1594] [Q&A] [answered]
I experiment with custom colours. First thing I tried is just:

    options.color_default      = 'black on white'  # the default fg and bg colors

but the result in *iTerm* for macOS is a darker grey background not a white or something similar´

![vd w-o-b in iTerm_Screen Shot 2022-11-13 at 3 09 28 pm](https://user-images.githubusercontent.com/1521852/201505362-92a9a683-caca-4a12-a664-d33c64bc9222.png)

Apple's default *Terminal* app doesn't look much different:

![vd w-o-b in Terminal_Screen Shot 2022-11-13 at 3 20 01 pm](https://user-images.githubusercontent.com/1521852/201505515-0b1cd48f-e6ec-48cb-adbb-be339cf50e55.png)

I know colors in terminal windows are a complicated matter, because of all the mapping which happens. So my question is:

Is this only me? Do I have somewhere in my terminal programs set some custom mappings which chance white to grey? Or is this a Visidata thing?

Any pointers are appreciated!

**Comment by saulpw** [comment, marked as answer]
@halloleo curses maps `white` to color 7, and that specific color would be set in your terminal theme.  You could change it there, or use a more specific color like `15` (for the brightest white) or any of the lighter grayscales from 250 to 255. Try the `open-colors` command and you can see the full list of available colors.

**Reply by halloleo** [reply]
Great. This is excellent! "black on 254" does a great jobs.
